doc/http-api/backend_swagger.yml

```diff
@@ -1,7 +1,7 @@
 swagger: '2.0'
 info:
   version: "1.0.0"
-  title: MongooseIM control backend REST API
+  title: "MongooseIM's backend REST API"
```

Comment:
@esl/mongooseim-developers: What does everyone think? It's either a `Control API` or an `Administration API` because they're **all** `Backend API`s (including the API for IM/chat clients)!